spec/integration/task_spec.cr

```diff
@@ -32,7 +32,7 @@ private def run(process, output = STDOUT)
 end
 
 private def task(task_name)
-  run("crystal spec/tasks.cr --no-debug -- #{task_name}")
+  run("crystal run --no-debug spec/tasks.cr -- #{task_name}")
```

Comment:
We've had this "Task --no-debug not found" showing up for a while, and I just found it here.